src/main.cpp

```diff
@@ -781,8 +781,12 @@ static ast::ASTContext buildListProgram(const std::string& search)
   auto ap = ast.make_node<ast::AttachPoint>(search, true, location());
   auto probe = ast.make_node<ast::Probe>(
       ast::AttachPointList({ ap }), nullptr, nullptr, location());
-  ast.root = ast.make_node<ast::Program>(
-      "", nullptr, ast::SubprogList(), ast::ProbeList({ probe }), location());
+  ast.root = ast.make_node<ast::Program>("",
+                                         nullptr,
+                                         std::vector<ast::MapDeclStatement*>{},
```

Comment:
Did you want to define a using alias for this type for consistent? E.g. `MapDeclList`? Then here you just have `ast::MapDeclList()`, which I think is cleaner.